Containers/nextcloud/entrypoint.sh

```diff
@@ -217,6 +217,8 @@ if ! [ -f "$NEXTCLOUD_DATA_DIR/skip.update" ]; then
         fi
 
         echo "Initializing nextcloud $image_version ..."
+        # TODO: touch rsync file to track what happens so that the container does not continue restarting? (it is not known what happens if it fails here at this step)
```

Comment:
@joshtrichards I need help with this. What should happen if we actually exit here? should we just touch a file so that we can track this and then report that rsync fails? or what should be done?